doc/source/modifiers.rst

```diff
@@ -52,6 +52,77 @@ on those modifiers can be found in the linked API documentation.
+with the :class:`~satpy.modifiers.parallax.ParallaxCorrectionModifier`
+API documentation.  Note that Satpy cannot apply modifiers to
+composites, so users wishing to apply parallax correction to a composite
+will have to use a lower level API or duplicate an existing composite
+recipe to use modified inputs.
+
+The parallax correction is directly calculated from the cloud top height.
+Information on satellite position is obtained from cloud top height
+metadata.  If no orbital parameters are present in the cloud top height
+metadata, Satpy will attempt to calculate orbital parameters from the
+platform name and start time.  The backup calculation requires skyfield
+and astropy to be installed.  If the metadata include neither orbital
+parameters nor platform name and start time, parallax calculation will
+fail.  Because the cloud top height metadata are used, it is essential
+that the cloud top height data are derived from the same platform as
+the measurements to be corrected are taken by.
+
+The parallax error moves clouds away from the observer.  Therefore, the
+parallax correction shifts clouds in the direction of the observer.  The
+space left behind by the cloud will be filled with fill values.  As the
+cloud is shifted toward the observer, it may occupy less pixels than before,
+because pixels closer to the observer have a smaller surface area.  It can
+also be deformed (a "rectangular" cloud may get the shape of a parallelogram).
+
+.. figure:: https://figshare.com/ndownloader/files/36422616/preview/36422616/preview.jpg
+   :width: 512
+   :height: 512
+   :alt: Satellite image without parallax correction.
+
+   SEVIRI view of southern Sweden, 2021-11-30 12:15Z, without parallax correction.
+   This is the ``natural_color`` composite as built into Satpy.
+
+
+.. figure:: https://figshare.com/ndownloader/files/36422613/preview/36422613/preview.jpg
+   :width: 512
+   :height: 512
+   :alt: Satellite image with parallax correction.
+
+   The same satellite view with parallax correction.  The most obvious change
+   are the gaps left behind by the parallax correction, shown as black pixels.
+   Otherwise it shows that clouds have "moved" south-south-west in the direction
+   of the satellite.  To view the images side-by-side or alternating, look at
+   `the figshare page <https://figshare.com/articles/figure/20211130121510-Meteosat-11-seviri-sswe-parallax_corrected_natural_color_jpg/20377203>`_
+
+The parallax correction is currently experimental and subject to change.
+Although it is covered by tests, there may be cases that yield unexpected
+or incorrect results.  It does not yet perform any checks that the
+provided (cloud top) height covers the area of the dataset for which
+the parallax correction shall be applied.
+
+.. versionadded:: 0.37
+
 A complete list can be found in the `etc/composites
 <https://github.com/pytroll/satpy/tree/main/satpy/etc/composites>`_
 source code and in the :mod:`~satpy.modifiers` module documentation.
```

Comment:
This is duplicated from above, should be removed.

Reply:
Fixed in 2c3a5d7a.